conda_build_all/builder.py

```diff
@@ -107,8 +113,15 @@ def select_lines(data, *args, **kwargs):
             # is only a suitable solution when selectors are also comments.
             return data
 
+        meta.final = False
```

Comment:
Is part of the multipass rendering process? Are we ok to change this or do we need to keep an eye out for changing/fixing this in the future.

Reply:
This is a bit of a hack.  Ideally, no one should be calling parse_again on their own.  Better to go through api.render.  To minimize the changes required to c-b-a, though, I left it in, but disable the final attribute, because the metadata is already finalized in conda-build's mind, and calling parse_again on it isn't good.  It's fine as a hack here, though.  It's kind of the same thing at the ResolvedDistribution does in conda-build-all: https://github.com/conda-tools/conda-build-all/blob/master/conda_build_all/resolved_distribution.py#L12

Reply:
Ideally we want to minimize c-b-a's hack count, but that is unlikely to be a perfectly linear process. Am ok with this one.